tests/test_e2e.py

```diff
@@ -156,13 +156,13 @@ def test_uri_device():
     device = json.loads(response_string)
     # Check json fields
     assert device.has_key("granularity")
-    assert device.has_key("maxDataAge")
-    assert device.has_key("macstr")
-    assert device.has_key("tankid")
+    assert device.has_key("max_data_age")
```

Comment:
No changes should be made to this file. This test is the reason for not making all the other array key changes.